.github/workflows/SDSGAS-CI.yaml

```diff
@@ -0,0 +1,23 @@
+name: SDSGAS-CI
```

Comment:
I feel like it might become cumbersome to prefix all of our actions with the repo name. Actions are per repo anyway, so there won't be any overlap. 

The same thing goes for the file name

Reply:
Good point, removed SDSGAS- from filename and workflow name